packages/asc-ui/src/components/Content/ContentStyle.ts

```diff
@@ -0,0 +1,13 @@
+import styled from '@datapunt/asc-core'
```

Comment:
Feels like bloating the codebase to, would make sense if we have default margins/paddings defined in the theme that we can pass

Reply:
refactored